### ISSUE TITLE
false positive `UnusedConstructorParams`

### ISSUE THREAD

**Issue by xuwei-k** [xuwei-k/scalafix-rules#2]
```scala
abstract class A(val x: Int)

class B(`type`: Int) extends A(`type`) // should not print warn
```

https://github.com/xuwei-k/scalafix-rules/blob/4ff568987b26c25218487a653d5474a500264f6e/rules/src/main/scala/fix/UnusedConstructorParams.scala
